apps/test/unit/templates/sectionProgress/StandardsViewHeaderButtonsTest.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../util/reconfiguredChai';
+import {StandardsViewHeaderButtons} from '../../../../src/templates/sectionProgress/standards/StandardsViewHeaderButtons';
+
+describe('StandardsViewHeaderButtons', () => {
+  it('opens lesson status dialog', () => {
+    const wrapper = mount(<StandardsViewHeaderButtons />);
+
+    wrapper
+      .find('Button')
+      .at(0)
```

Comment:
Can you look for these buttons based on the ids you gave them?

Reply:
That wasn't working. It was saying it was finding two things for those ids. So I'm going to remove the ids and just go with this since it was working